applications/schema.py

```diff
@@ -200,7 +200,7 @@ def mutate(self, info, **kwargs):
         )
 
 
-class Mutation(graphene.ObjectType):
+class Mutation:
```

Comment:
Is the inheritance unnecessary? At least it serves as clarification `graphene.ObjectType` vs `graphene.Mutation`

Reply:
I removed it on this because [this](https://github.com/City-of-Helsinki/berth-reservations/blob/develop/resources/schema.py#L691) one didn't have it, and that one was taken from Kukkuu.

I was talking about it with @igordavydsson and came to the conclusion that, since it's already extended on the main schema, there's no need to extend it twice.
```python
class Mutation(resources.schema.Mutation, graphene.ObjectType):
    pass

new_schema = build_schema(query=Query, mutation=Mutation, types=extended_types)
```

Reply:
yeah, it seems that this aggregators for mutations and queries are kinda like mixin objects, and do not have to inherit from `graphene.ObjectType`. would be nice to make it consistent across all apps in a separate commit.